diff --git a/node_modules/wrangler/wrangler-dist/cli.js b/node_modules/wrangler/wrangler-dist/cli.js
index 8da7c4f..1423bc0 100644
--- a/node_modules/wrangler/wrangler-dist/cli.js
+++ b/node_modules/wrangler/wrangler-dist/cli.js
@@ -182135,7 +182135,7 @@ async function captureGlobalException(e10) {
 }
 async function closeSentry() {
   {
-    await close();
+    // await close();
   }
 }
 var import_undici11, sentryReportingAllowed, makeSentry10Transport, disabledDefaultIntegrations;
