diff --git a/node_modules/wrangler/wrangler-dist/cli.js b/node_modules/wrangler/wrangler-dist/cli.js
index 7dd3ec0..23e7260 100644
--- a/node_modules/wrangler/wrangler-dist/cli.js
+++ b/node_modules/wrangler/wrangler-dist/cli.js
@@ -230498,7 +230498,7 @@ async function captureGlobalException(e9) {
 }
 async function closeSentry() {
   {
-    await close2();
+    // await close2();
   }
 }
 var import_undici14, sentryReportingAllowed, makeSentry10Transport, disabledDefaultIntegrations;
